lib/plugins/parse5-flatten-script.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const {resolve} = require('url');
+const logger = require('bole')('posthtml-flatten-script');
```

Comment:
Your logger name isn't matching the rest of collapsify.

Reply:
Fixed (decided to log to collapsers:html group).